MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -159,6 +159,13 @@ public GraphicsAdapter Adapter
             private set;
         }
 
+        internal GraphicsMetrics _graphicsMetrics;
+
+        /// <summary>
+        /// The rendering information for debugging and profiling. 
+        /// </summary>
+        public GraphicsMetrics GraphicsMetrics { get { return _graphicsMetrics; } set { _graphicsMetrics = value; } }

```

Comment:
Maybe this should be just `GraphicsDevice.Metrics`.  Also the docs could be better... see my comments below.